tasks/beats-Windows.yml

```diff
@@ -0,0 +1,57 @@
+---
+- set_fact: beat_name="{{ beat }}"
+- name: Windows - Ensure WinPcap is installed before install packetbeat
+  win_chocolatey:
+    name: nmap
+    state: present
+    version: 7.12
+  when: beat_name == "packetbeat"
+
+- name: Windows - Ensure {{ beat }} is installed
```

Comment:
Hi,
First of all thank you for providing installation playbook, but as I running with your playbook on windows, I got errors which says:
*The service did not respond to the start or control request in a timely fashion. ERROR 1053*
*env:* .  
*beat: metricbeat 7.0.0* .   
*os: windows server 2016* . 
*ansible: lastest stable version* . 
however,  I found that the win_chocolatey way to install will not using the configuration I predefined, which make that error happen?(not sure.)
I changed the config **use_repository** which working good(PS: fixed an error, will comment on code) .  
do you have a test for metricbeat? (this might be not your issue, I just met this error for using this code snippet, you might help me)

Reply:
sorry,I'm late. use_repository=true works for me , u can post comment to https://chocolatey.org/packages/metricbeat ,or paste your error logs